config/mail.global.php.dist

```diff
@@ -128,6 +128,7 @@ return [
          * All of them will be capable of sending any of the previously configured emails.
          */
         'mail_services' => [
+            'throw_on_cancel' => false,
```

Comment:
I just realized I wasn't clear enough during our discussion in the ticket :sweat_smile:. Sorry, my bad.

I meant that the config should be defined on a per-service basis, not globally for every service.

While it's true that in most of the cases you want the same behavior in every service, every entry inside `acmailer.mail_service` should be the definition of a service, and nothing else, as otherwise, it could lead to confusions.

For example, with this configuration, you could try to fetch the `acmailer.mail_service.throw_on_cancel` service.

Also, you can still define the config once and then make the rest of the services extend from that one.